Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ListViewExtensions/ListViewExtensionsPage.xaml.cs

```diff
@@ -10,19 +12,61 @@ namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
 {
     public sealed partial class ListViewExtensionsPage : Page, IXamlRenderListener
     {
+        private ListView sampleListView;
+        private TextBlock indexInput;
+        private TextBlock itemPlacementInput;
+        private CheckBox disableAnimationInput;
+        private CheckBox scrollIfVisibileInput;
+        private TextBlock additionalHorizontalOffsetInput;
+        private TextBlock additionalVerticalOffsetInput;
+
         public ListViewExtensionsPage()
         {
             this.InitializeComponent();
+            Load();
         }
 
-        public async void OnXamlRendered(FrameworkElement control)
+        public void OnXamlRendered(FrameworkElement control)
         {
-            var sampleListView = control.FindChildByName("SampleListView") as ListView;
+            sampleListView = control.FindChildByName("SampleListView") as ListView;
```

Comment:
Wish I could have built and seen this, trying to think if there's a better way we can show this sample. It almost seems as if we want a static page vs. using the property panel...?